<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>201</y>
      <width>530</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color:rgb(226, 226, 226);

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_icon_top_bar">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>2</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
background-image:  url(F:/books/agriculture project/desktop app/agri/watering-can.png);
background-position: center;
background-repeat: no-repeat;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="title_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>7</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
      </font>
     </property>
     <property name="text">
      <string>Agriculture Irrigation Desktop App | CARE IUB | Add New Crop</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_16">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color:rgb(226, 226, 226);

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="closeButton_2">
      <property name="geometry">
       <rect>
        <x>1304</x>
        <y>0</y>
        <width>37</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {	
	border: none;
	background-color: transparent;

background-image: url(F:/books/agriculture project/desktop app/agri/icons8-close-window-20.png);

background-repeat: no-repeat;
background-position: center;
border-radius: 0px;

}
QPushButton:hover {
	background-color: rgb(255, 82, 76);
border-radius: 0px;
}
QPushButton:pressed {	
	background-color:rgb(255, 82, 76);
border-radius: 0px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="minimize_btn_2">
      <property name="geometry">
       <rect>
        <x>1267</x>
        <y>0</y>
        <width>37</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {	
	border: none;
	background-color: transparent;

background-image: url(F:/books/agriculture project/desktop app/agri/icons8-minimize-window-20.png);

background-repeat: no-repeat;
background-position: center;
border-radius: 0px;

}
QPushButton:hover {
	background-color:rgb(191, 191, 191);
border-radius: 0px;
}
QPushButton:pressed {	
	background-color:rgb(162, 162, 162);
border-radius: 0px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="minimize_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>37</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {	
	border: none;
	background-color: transparent;

background-image: url(F:/books/agriculture project/desktop app/agri/icons8-minimize-window-20.png);

background-repeat: no-repeat;
background-position: center;
border-radius: 0px;

}
QPushButton:hover {
	background-color:rgb(191, 191, 191);
border-radius: 0px;
}
QPushButton:pressed {	
	background-color:rgb(162, 162, 162);
border-radius: 0px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>67</x>
        <y>0</y>
        <width>37</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {	
	border: none;
	background-color: transparent;

background-image: url(F:/books/agriculture project/desktop app/agri/icons8-close-window-20.png);

background-repeat: no-repeat;
background-position: center;
border-radius: 0px;

}
QPushButton:hover {
	background-color: rgb(255, 82, 76);
border-radius: 0px;
}
QPushButton:pressed {	
	background-color:rgb(255, 82, 76);
border-radius: 0px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>532</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: rgb(246, 247, 249);
color:rgb(30, 25, 83);
border: 1px solid rgb(163, 163, 163);
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>31</y>
       <width>528</width>
       <height>359</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
border:none;
}

QPushButton {
	border: 1px solid rgb(229, 229, 234);
	border-radius: 5px;	
	background-color:rgb(255, 255, 255);
color:rgb(30, 25, 83);
}
QPushButton:hover {
	
	border: 2px solid rgb(229, 229, 234);
}
QPushButton:pressed {	
	background-color:rgb(234, 234, 229);
border: 2px solid  rgb(234, 234, 229);
}



QComboBox{
	background-color: rgb(255, 255, 255);
	border-radius: 5px;
	border: 1px solid rgb(229, 229, 234);
	padding: 5px;
	padding-left: 10px;

}
QComboBox:hover{
	border: 2px solid  rgb(229, 229, 234);
}
QComboBox::drop-down {
image : url(F:/books/gui - Copy/Youtube Downloader/asd.png);
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px; 
	
	border-left-width: 1px;
	border-left-color: rgb(229, 229, 234);
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;	
	
	background-position: center;
	background-repeat: no-reperat;
 }
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_32">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>529</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color:rgb(255, 255, 255);

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_48">
       <property name="geometry">
        <rect>
         <x>49</x>
         <y>14</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins SemiBold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add New Crop Data</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>281</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius:10px;


}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="city_comboBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>156</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>66</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
       <property name="accessibleDescription">
        <string>sadf</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit{
border: 1px solid rgb(229, 229, 234);
border-radius: 5px;
color: rgb(30, 25, 83)
}

QDateEdit::drop-down {
image : url(F:/books/gui - Copy/Youtube Downloader/asd.png);
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px; 
	
	border-left-width: 1px;
	border-left-color: rgb(229, 229, 234);
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;	
	
	background-position: center;
	background-repeat: no-reperat;
 }</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>165</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
        </font>
       </property>
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>116</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
        </font>
       </property>
       <property name="text">
        <string>Soil Type:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>26</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
        </font>
       </property>
       <property name="text">
        <string>Crop:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>74</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
        </font>
       </property>
       <property name="text">
        <string>Sowing Date:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="crop_comboBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(30, 25, 83)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="soil_comboBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="add_crop_cnfrm">
       <property name="geometry">
        <rect>
         <x>158</x>
         <y>220</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add crop</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="cancel_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </widget>
     <zorder>frame_4</zorder>
     <zorder>frame_32</zorder>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
